includes/drush/drushrc.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @file
+ * Custom overrides.
+ */
+
+if (str_ends_with(getenv('HOSTNAME'), '-acc') || 
+    str_ends_with(getenv('HOSTNAME'), '-prod')) {
+  $options['l'] = end(explode(",", getenv('VIRTUAL_HOST')));
+  $options['r'] = getenv('DOCUMENT_ROOT');
+}
```

Comment:
Maybe this can be simplified to remove HOSTNAME need.
```
// Specify the base_url that should be used when generating links.
if (isset(getenv('VIRTUAL_HOST'))){
  $options['l'] = end(explode(",", getenv('VIRTUAL_HOST')));
}

// Specify your Drupal core base directory.
if (isset(getenv('DOCUMENT_ROOT'))){
  $options['r'] = getenv('DOCUMENT_ROOT');
} 
```